.travis.yml

```diff
@@ -4,52 +4,45 @@ dist: trusty
 
 matrix:
   include:
+    - name: Lint
```

Comment:
Have you seen Travis' [build stages](https://docs.travis-ci.com/user/build-stages/) before? I think they might be better suited to this task. Rather than hack in testing with environment variables in the matrix, we can just create `lint`, `code coverage`, and `test` stages and run them conditionally.  What do you think?

Reply:
reorganized to use stages 👍